src/components/structures/LoggedInView.js

```diff
@@ -353,7 +354,9 @@ const LoggedInView = React.createClass({
 
         let topBar;
         const isGuest = this.props.matrixClient.isGuest();
-        if (this.props.hasNewVersion) {
+        if (this.props.showCookieBar) {
+            topBar = <CookieBar />;
```

Comment:
Annoying that this will prevent the display of the password nag bar. I don't have a fix to suggest for this though, :/